README.md

```diff
@@ -256,8 +256,8 @@ questions and get our support.
 [ci-url]: https://github.com/opencv/cvat/actions
 [gitter-img]: https://img.shields.io/gitter/room/opencv-cvat/public?style=flat
 [gitter-url]: https://gitter.im/opencv-cvat
-[coverage-img]: https://coveralls.io/repos/github/cvat-ai/cvat/badge.svg?branch=develop
-[coverage-url]: https://coveralls.io/github/cvat-ai/cvat?branch=develop
+[coverage-img]: https://codecov.io/github/opencv/cvat/branch/develop/graph/badge.svg?token=SC0BwtMJXv
+[coverage-url]: https://codecov.io/github/opencv/cvat
```

Comment:
I think it would be perfect to have two different badges: server and UI. Can it be done easily with codecov?

Reply:
We cannot get several badges with codecov. But we can get code coverage by custom components. We can see it in the PR comment.

Reply:
Yes, I saw it in the first version of this PR, but now I see this:
![Screenshot from 2023-06-06 20-43-14](https://github.com/opencv/cvat/assets/43179655/9f795c3c-35e5-4667-9b2e-0bb2f1c482cd)
Do you know the reason for this error?

Reply:
We don't have base commit now. We should merge it into develop. After that, we can receive reports on these links in PR. Now you may see the report [here](https://app.codecov.io/gh/opencv/cvat/tree/ay%2Fadd-codecov).